docs/pages/blog/introducing-toolpad-core.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Introducing Toolpad Core
```

Comment:
Would "Introducing" feel too much for something that is stable? For example, I reframed https://mui.com/blog/introducing-pigment-css/ as a "preview" as it felt more accurate to what it is.